README.md

```diff
@@ -25,3 +23,28 @@ You will need to switch workspace to plan/apply in a particular environment:
 > terraform workspace select stage
 Switched to workspace "stage".
 ```
+
+### Continuous Integration
+
+This project builds in Buildkite where there are 3 pipelines to cover build and deployment through stage and production environments.
```

Comment:
Three pipelines but only two environments? This sentence makes me think the two numbers are related, but maybe they're not?

Reply:
There's one for build and two for deployment. Probably worth clarifying here.